package.json

```diff
@@ -46,6 +46,28 @@
     "gruntplugin"
   ],
   "dependencies": {
-    "grunt": "~0.4.5"
-  }
+    "grunt": "~0.4.5",
+    "js2coffee": "^0.3.1"
+  },
+  "_id": "grunt-translate-compile@0.1.6",
+  "dist": {
+    "shasum": "13861cba07bffe5f65d24bf0eb278261ce9b52b7",
+    "tarball": "http://registry.npmjs.org/grunt-translate-compile/-/grunt-translate-compile-0.1.6.tgz"
+  },
+  "_from": "grunt-translate-compile@*",
+  "_npmVersion": "1.3.26",
+  "_npmUser": {
+    "name": "cainaf",
+    "email": "cainafsantos@gmail.com"
+  },
+  "maintainers": [
+    {
+      "name": "cainaf",
+      "email": "cainafsantos@gmail.com"
+    }
+  ],
+  "directories": {},
+  "_shasum": "13861cba07bffe5f65d24bf0eb278261ce9b52b7",
+  "_resolved": "https://registry.npmjs.org/grunt-translate-compile/-/grunt-translate-compile-0.1.6.tgz",
```

Comment:
Hello, I'm new to using npm, could you tell me what are these `_<var>` used for?